tests/api/api_config.py

```diff
@@ -44,14 +44,14 @@
         "/augsburg/de/wp-json/extensions/v3/pages/",
         "tests/api/expected-outputs/augsburg_de_pages.json",
         200,
-        5,
+        7,
```

Comment:
```suggestion
        5,
```
You can cut down those queries by selecting the related icon objects in the API.

Replace `region.pages.select_related("organization")` by `region.pages.select_related("organization__icon")` in `integreat_cms/api/v3/pages.py` and `.select_related("category", "organization")` by `.select_related("category", "organization__icon")` in `integreat_cms/api/v3/locations.py`.